### DISCUSSION TITLE
How to change the OAuth2 Scope for discord authentication

### DISCUSSION THREAD

**Discussion by krishnerkar** [supabase/supabase#3547] [Questions] [answered]
I'm using Supabase UI to authenticate with discord, and i was playing around with the nextjs-auth example, i couldn't figure out how to change the authentication scopes for discord OAuth2

![image](https://user-images.githubusercontent.com/49949733/137478308-d15ba90d-3b9e-4e15-b26f-2610dbae930b.png)

**Comment by inian** [comment, marked as answer]
Cross posted here https://github.com/supabase/supabase/issues/3548

**Comment by awalias** [comment]
I think you want the `scopes` parameter here? https://supabase.io/docs/reference/javascript/auth-signin#parameters
